app/domain/authentication/authn_gce/validate_resource_restrictions.rb

```diff
@@ -0,0 +1,66 @@
+module Authentication
+  module AuthnGce
+
+    # This class is responsible of validating resource restrictions which are configured on a Conjur host or user
```

Comment:
we decided to support user as well? was this checked manually?

Reply:
This class is responsible for *

Reply:
we won't document this but it is tested in an automated test.